### PR TITLE
Bump and pin requirements

### PR DESCRIPTION
**Pull request by MartinHjelmare**
- numpy needs to stay compatible with latest tensorflow version which has constraints on the numpy version required. Make sure to check this before upgrading numpy.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,6 @@
-imjoy-rpc>=0.2.56
-python-socketio[asyncio_client]
-pyyaml
-aiohttp
-aiohttp_cors
+imjoy-rpc==0.2.72
```
Comment (MartinHjelmare) [inline requirements.txt:1]: Is this imjoy-rpc version compatible with master branch?

Comment (MartinHjelmare) [reply requirements.txt:1]: Let's keep this for now at 0.2.56. It'll be bumped in another PR.
```suggestion
imjoy-rpc==0.2.56
```

File: requirements.txt [hunk 2]
```diff
@@ -1,5 +1,6 @@
...
+python-socketio[asyncio_client]==5.2.1
```
Comment (MartinHjelmare) [inline requirements.txt:2]:
```suggestion
python-socketio[asyncio_client]==5.0.4  # pin for now to solve unknown connection issue
```

**Comment by oeway** [comment]
Looks good!

**Pull request merged**